### PR TITLE
onboarding terraform-redhat/terraform-aws-rosa-sts

### PR DESCRIPTION
**Pull request by AsherShoshan**

### CODE REVIEW

File: ci-operator/config/terraform-redhat/terraform-aws-rosa-sts/terraform-redhat-terraform-aws-rosa-sts-main.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: unit
+  commands: make test-unit
```
[the remark this answers is not in the document]
Comment (AsherShoshan) [reply ci-operator/config/terraform-redhat/terraform-aws-rosa-sts/terraform-redhat-terraform-aws-rosa-sts-main.yaml:15]: so any unit test script there?

[the remark this answers is not in the document]
Comment (AsherShoshan) [reply ci-operator/config/terraform-redhat/terraform-aws-rosa-sts/terraform-redhat-terraform-aws-rosa-sts-main.yaml:15]: what unit test script is there.  this repo is almost empty and contains only .tf files. so what's the point of onboarding it.

[the remark this answers is not in the document]
Comment (sagidayan) [reply ci-operator/config/terraform-redhat/terraform-aws-rosa-sts/terraform-redhat-terraform-aws-rosa-sts-main.yaml:15]: Currently, nothing.

Maybe you can start with tf lint, or something. just to have a test.

**Comment by AsherShoshan** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-aws-rosa-sts-main-unit

**Pull request merged**